templates/frontend/objects/article_details.tpl

```diff
@@ -259,6 +260,7 @@
 			{/if}
 			{if $supplementaryGalleys}
 				<div class="item galleys">
+					<h3 class="pkp_screen_reader">Additional Files</h3>
```

Comment:
This (and "Downloads" above) need to be translatable strings that use `{translate key="..."}`.

Reply:
I'm not sure about these namings, do you think `submission.layout.galley.header` and `submission.layout.galley.additionalHeader` work?

Reply:
I like to keep it simple. Try `submission.downloads` and `submission.additionalFiles`.

Reply:
Try now? @NateWr